core/src/bank_forks.rs

```diff
@@ -188,6 +188,7 @@ impl BankForks {
         let descendants = self.descendants();
         self.banks
             .retain(|slot, _| descendants[&root].contains(slot));
+        self.confidence.retain(|slot, _| slot == &root || descendants[&root].contains(slot));
```

Comment:
did this change get removed in the latest commit?

Reply:
No, it just got reformatted in the latest commit ;)